frontend/src/components/SearchLozenge/SearchLozenge.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './SearchLozenge.css';
+
+const SearchLozenge = (props) => {
+  let { searchTerm } = props;
+  if (!searchTerm) {
+    searchTerm = '';
+  }
+  const lozenges = searchTerm.split(' ');
+  return (
+    <div className="searchLozenge container">
+      <div className="row">
+        <div className="lozengeBubbles">
+          {lozenges.map(lozenge => (
+            <div className="lozenges">
+              {lozenge.charAt(0).toUpperCase() }
+              {lozenge.substr(1)}
```

Comment:
I think maybe creating a capitalize function and calling it here would be clearer, like
```
const capitalize = (s) => {
  return s.charAt(0).toUpperCase() + s.slice(1).toLowerCase();
}
```